templates/vue/src/components/TapestryMap.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div>
+    <div style="height: 800px; width: 100%">
+      <l-map
+        :options="mapOptions"
+        :bounds="setBounds"
+        :max-bounds="worldBounds"
+        style="height: 80%"
+        @update:center="updateCenter"
+        @update:zoom="updateZoom"
+      >
+        <l-tile-layer :url="url" :attribution="attribution" />
+        <l-marker
+          v-for="marker in markerlocations"
+          :key="marker.id"
+          :lat-lng="marker.pos"
+          :icon="marker.access ? icon : inaccessibleIcon"
```

Comment:
Can you also add a separate icon for draft nodes? It would be nice if it follows the look and feel of draft nodes (i.e. dashed outline)